### PR TITLE
feat(hardware definitons): add FCC and IC fields

### PR DESCRIPTION
**Pull request by marvinmarnold**
**Issue**

- Link: https://github.com/NebraLtd/Hotspot-Production-Tool/issues/12
- Summary: Support different FCC and ICC IDs on different devices.

**How**
- Added FCC and ICC ids to variant_defintions.
- Only set values for variants that are currently explicitly
  handled in HPT: https://github.com/NebraLtd/Hotspot-Production-Tool/blob/d948acc4245cbe66777cae26660789b249cbb349/app/main.py#L90
- For future proofing, made FCC_IDS an array even though we currently only seem to have one
value per device type.

**References**
- Related to: https://github.com/NebraLtd/Hotspot-Production-Tool/issues/34
- Related to: https://github.com/NebraLtd/Hotspot-Production-Tool/issues/11
- Related to: https://github.com/NebraLtd/Hotspot-Production-Tool/issues/10

**Checklist**

- [x] Tests added
- [x] Cleaned up commit history (rebase!)
- [ ] Documentation added
- [x] Thought about variable and method names

### CODE REVIEW

**Review by vpetersson — APPROVED** [review]
Nice fix! Full disclosure: I didn't double check the values of the FCC_IDs etc but looks clean.

**Comment by vpetersson** [comment]
@marvinmarnold see https://github.com/NebraLtd/Hotspot-Production-Tool/issues/34#issuecomment-989184286

**Comment by shawaj** [comment]
you beat me to it @vpetersson haha

File: hm_pyhelper/hardware_definitions.py [hunk 1]
```diff
@@ -38,7 +38,10 @@ def is_raspberry_pi():
         'BUTTON': 26,
         'ECCOB': True,
         'TYPE': 'Full',
-        'CELLULAR': False
+        'CELLULAR': False,
+        'FCC_IDS': ['2AZDM-HNTIN'],
+        'CONTAINS_FCC_IDS': ['2AHRD-EPN8531', '2AB8JCSR40', '2ARPP-GL5712UX'],
+        'ICC_IDS': ['27187-HNTIN']
```
Comment (shawaj) [inline hm_pyhelper/hardware_definitions.py:44]: should be IC or ISED really

File: hm_pyhelper/hardware_definitions.py [hunk 2]
```diff
@@ -94,8 +108,12 @@ def is_raspberry_pi():
         'BUTTON': 23,
         'ECCOB': True,
         'TYPE': 'Full',
-        'CELLULAR': False
+        'CELLULAR': False,
+        'FCC_IDS': ['2AZDM-HNTLGT'],
```
Comment (shawaj) [inline hm_pyhelper/hardware_definitions.py:112]: Need to update these - one sec will get the details

**Comment by shawaj** [comment]
at the moment, we are only certifying the SPI ethernet based light gateway, not the USB one https://github.com/NebraLtd/Hotspot-Production-Tool/issues/11#issuecomment-989193692

File: hm_pyhelper/hardware_definitions.py (hunk 1 above)
Comment (marvinmarnold) [reply hm_pyhelper/hardware_definitions.py:44]: Don't know this type TBH, is this right? https://www.ic.gc.ca/eic/site/ceb-bhst.nsf/eng/h_tt00051.html#Q10

File: hm_pyhelper/hardware_definitions.py (hunk 2 above)
Comment (marvinmarnold) [reply hm_pyhelper/hardware_definitions.py]: Updated: https://github.com/NebraLtd/hm-pyhelper/pull/80/files#diff-2316c9dbb9d32df1562699aec0829c762a67684f74c5573db939f8ec1d5d7858R115

**Comment by marvinmarnold** [comment]
@shawaj mind double checking these values?

File: hm_pyhelper/hardware_definitions.py (hunk 1 above)
Comment (shawaj) [reply hm_pyhelper/hardware_definitions.py]: yep lets go with IC

**Comment by shawaj** [comment]
> @shawaj mind double checking these values?

checking now

**Comment by shawaj** [comment]
looks good to me @marvinmarnold

**Review by shawaj — APPROVED** [review]

**Pull request merged**